src/components/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FunctionComponent } from 'react';
+
+import classNames from 'classnames';
+
+export interface SpinnerProps {
+	size?: 'large';
```

Comment:
![image](https://user-images.githubusercontent.com/1710840/58538758-196ca800-81f6-11e9-8ac7-51cdfdc05c5e.png)
required should be set to false?
and maybe default: "small"?

Reply:
Die proptypes zijn autogenerated uit de TS Props. Er is geen small variant, dus lijkt mij vreemd om daar waarde aan toe te kennen die niks gaat doen met de component.